File: scripts/yao_node.py
#!/usr/bin/env python3
#coding=utf-8

import rospy
from std_msgs.msg import String

if __name__ == "__main__":
    rospy.init_node("yao_node")
    rospy.logwarn("初始化yao_node成功")
    pub = rospy.Publisher("gie_gie",String,queue_size=10)

    rate = rospy.Rate(10)
    while not rospy.is_shutdown():
        rospy.loginfo("我要开始刷屏了！")
        msg = String()
        msg.data = "求上车++！"
        pub.publish(msg)
        rate.sleep()

File: src/yao_node.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main(int argc, char  *argv[])
{
    ros::init(argc,argv,"yao_node");
    printf("瑶的节点\n");

    ros::NodeHandle nh;
    ros::Publisher pub=nh.advertise<std_msgs::String>("QiuDai",10);
    while (ros::ok())
    {
        printf("我要刷屏了。\n");
        ros::Rate loop_rate(10);
        std_msgs::String msg;
        msg.data = "上车++！\n";
        pub.publish(msg);
        loop_rate.sleep();

    }
    
    return 0;
}
